### PR TITLE
issue #36_sbi_spec_3.6,3.7

### PR DESCRIPTION
**Pull request by hassanraza164**
# Pull Request Template

## Prequisites

- [X] Did you run lint -cf `filename` on your changes, if you are commiting a .c file?
- [] Did you run lint -sf `filename` on your changes, if you are commiting a .sh file?
- [ ] Does your code compile properly using `make`?
- [X] Did you related issue number in the description?

## Description

issue #36_sbi_spec_3.6,3.7. test cases have been written.
## Technical Debt

<Write about any missing things that you are going to do in the future related to this PR>

### CODE REVIEW

File: code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+int test_case(void){
+    struct sbiret get_marchid = sbi_get_marchid();
+
+    if (get_marchid.error != SBI_SUCCESS)
+      exit_test(2);
+    
+    if (get_marchid.value == 0)
+       exit_test(TEST_PASS);   
+}
```
Comment (mmhus) [inline code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c]: missing line at end of file

File: code/src/test_srcs/test__sbi_get_mimpid/test__sbi_get_mimpid.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+int test_case(void){
+    struct sbiret get_mimpid = sbi_get_mimpid();
+
+    if (get_mimpid.error != SBI_SUCCESS)
+      exit_test(TEST_FAIL);
+    
+    if (get_mimpid.value == 0)
+       exit_test(TEST_PASS);   
+}
```
Comment (mmhus) [inline code/src/test_srcs/test__sbi_get_mimpid/test__sbi_get_mimpid.c]: missing line at EOF.

File: code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#include "test_macros.h"
```
Comment (mmhus) [inline code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c]: Are you facing any issues running spike?

**Review by mmhus — CHANGES_REQUESTED** [review]

File: code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c (hunk 3 above)
Comment (hassanraza164) [reply code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c]: Now, it is running

**Pull request closed without merging**